### PR TITLE
Check for aws config all regions

### PR DESCRIPTION
**Pull request by livnoni**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: checkov/terraform/checks/resource/aws/ConfigConfgurationAggregatorAllRegions.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        :param conf: aws_config_configuration_aggregator configuration
+        :return: <CheckResult>
+        """
+        self.evaluated_keys = ["account_aggregation_source", "organization_aggregation_source"]
+
+        if "account_aggregation_source" in conf:
+            current_regions = flat_list(conf.get("account_aggregation_source", {})[0].get("regions"))
+            regions = ["us-east-2", "us-east-1", "us-west-1", "us-west-2"]  # do i need to set all of the regions?
+            if set(current_regions) == set(regions):
+                return CheckResult.PASSED
```
Comment (nimrodkor) [inline checkov/terraform/checks/resource/aws/ConfigConfgurationAggregatorAllRegions.py:35]: @livnoni Let's always look for `all_regions = true`, relevant for account and organizational aggregation sources
image

**Review by nimrodkor — COMMENTED** [review]
@livnoni Nice!
What happens if the configurations aren't present, as both are optional?

File: checkov/terraform/checks/resource/aws/ConfigConfgurationAggregatorAllRegions.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        if "account_aggregation_source" in conf:
+            all_regions_attr = conf.get("account_aggregation_source", {})[0].get("all_regions")
+            if all_regions_attr:
+                return CheckResult.PASSED
+            return CheckResult.FAILED
+        if "organization_aggregation_source" in conf:
+            if conf.get("organization_aggregation_source", {})[0].get("all_regions"):
+                return CheckResult.PASSED
+            return CheckResult.FAILED
+
```
Comment (nimrodkor) [inline checkov/terraform/checks/resource/aws/ConfigConfgurationAggregatorAllRegions.py:32]: @livnoni There's a case where both `ifs` don't match - let's return CheckResult.FAILED in that case

**Review by nimrodkor — APPROVED** [review]

File: checkov/terraform/checks/resource/aws/ConfigConfgurationAggregatorAllRegions.py (hunk 2 above)
Comment (livnoni) [reply checkov/terraform/checks/resource/aws/ConfigConfgurationAggregatorAllRegions.py:32]: got it

Comment (livnoni) [reply checkov/terraform/checks/resource/aws/ConfigConfgurationAggregatorAllRegions.py]: I returned FAILED + added this scenario to the UT.
Thanks @nimrodkor

**Pull request merged**